javascript-arrays/main.js

```diff
@@ -0,0 +1,22 @@
+var colors = ['red', 'white', 'blue'];
+console.log('value of colors[0]:', colors[0]);
+console.log('value of colors[1]:', colors[1]);
+console.log('value of colors[2]:', colors[2]);
+console.log('America is' + ' ' + colors[0] + ',' + ' ' + colors[1] + ',' + ' ' + colors[2] + '.');
```

Comment:
just a note: you can do something like `'America is ' + colors[0]` so that you don't have to concatenate the extra space.

Reply:
got it